Dockerfile

```diff
@@ -2,18 +2,18 @@
 #
 # BUILD: docker build --no-cache --rm -t nordaaker/convos .
 # RUN:   docker run -it --rm -p 8080:3000 -v /var/convos/data:/data nordaaker/convos
-FROM alpine:3.4
+FROM alpine:3.5
 MAINTAINER jhthorsen@cpan.org
 
-WORKDIR /
+ARG port=3000
```

Comment:
It create a build-time variable that the user can modify when building his image.